File: 02.TechModule-09.2017/SoftwareTech/Exams/MyExam/C#-Skeleton/ProjectRider/Models/Project.cs
using System.ComponentModel.DataAnnotations;

namespace ProjectRider.Models
{
    public class Project
    {
        [Key]
        public int Id { get; set; }

        [Required]
        public string Title { get; set; }

        [Required]
        public string Description  { get; set; }

        public long Budget { get; set; }
    }
}

File: 02.TechModule-09.2017/SoftwareTech/Exams/MyExam/Java-Skeleton/src/main/java/projectrider/entity/Project.java
package projectrider.entity;

import javax.persistence.*;

@Entity
@Table(name = "projets")
public class Project {
    //TODO: Implement me ...
    @Id
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    private int id;

    @Column(nullable = false)
    private String title;

    @Column(nullable = false)
    private String description;

    @Column(nullable = false)
    private long budget;

    public int getId() {
        return id;
    }

    public void setId(int id) {
        this.id = id;
    }

    public String getTitle() {
        return title;
    }

    public void setTitle(String title) {
        this.title = title;
    }

    public String getDescription() {
        return description;
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public long getBudget() {
        return budget;
    }

    public void setBudget(long budget) {
        this.budget = budget;
    }
}
